src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonTypeInfo.cs

```diff
@@ -167,7 +167,7 @@ internal JsonTypeInfo()
             Debug.Assert(false, "This constructor should not be called.");
         }
 
-        internal JsonTypeInfo(Type type, JsonSerializerOptions options!!, bool dummy)
+        internal JsonTypeInfo(Type type, JsonSerializerOptions options!!)
```

Comment:
Can we remove this constructor altogether? IIRC all instantiation paths for JsonTypeInfo require `JsonConverter` one way or another.